db/migrate/20150219194157_remove_location_from_stations.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveLocationFromStations < ActiveRecord::Migration
+  def change
+    remove_column :stations, :location, :string
```

Comment:
`location` ends up being the same information provided in each station objects `address` column. Realistically, I guess I should include a method in the rake task to make sure no `address` columns are `null`.